src/gsy_e/models/strategy/energy_parameters/virtual_heat_pump.py

```diff
@@ -0,0 +1,196 @@
+import logging
+from typing import Optional, Union, Dict
+
+import sympy as sp
+from gsy_framework.constants_limits import ConstSettings, GlobalConfig
+from gsy_framework.read_user_profile import InputProfileTypes
+from gsy_framework.utils import convert_W_to_kWh, convert_kWh_to_W
+from pendulum import DateTime
+
+from gsy_e.constants import FLOATING_POINT_TOLERANCE
+from gsy_e.models.strategy.energy_parameters.heat_pump import (
+    HeatPumpEnergyParametersBase, WATER_DENSITY)
+from gsy_e.models.strategy.profile import EnergyProfile
+
+logger = logging.getLogger(__name__)
+
+CALIBRATION_COEFFICIENT = 0.85
+WATER_SPECIFIC_TEMPERATURE = 4182
+GROUND_WATER_TEMPERATURE_C = 12
+
+
+class VirtualHeatpumpEnergyParameters(HeatPumpEnergyParametersBase):
+    """Energy parameters for the virtual heatpump strategy class."""
+    # pylint: disable=too-many-arguments
+    def __init__(
+            self,
+            maximum_power_rating_kW: float = ConstSettings.HeatPumpSettings.MAX_POWER_RATING_KW,
+            min_temp_C: float = ConstSettings.HeatPumpSettings.MIN_TEMP_C,
+            max_temp_C: float = ConstSettings.HeatPumpSettings.MAX_TEMP_C,
+            initial_temp_C: float = ConstSettings.HeatPumpSettings.INIT_TEMP_C,
+            tank_volume_l: float = ConstSettings.HeatPumpSettings.TANK_VOL_L,
+            water_supply_temp_C_profile: Optional[Union[str, float, Dict]] = None,
+            water_supply_temp_C_profile_uuid: Optional[str] = None,
+            water_return_temp_C_profile: Optional[Union[str, float, Dict]] = None,
+            water_return_temp_C_profile_uuid: Optional[str] = None,
+            dh_water_flow_m3_profile: Optional[Union[str, float, Dict]] = None,
```

Comment:
IIRC, this is given in m^3/h, so how about:
```suggestion
            dh_water_flow_m3_h_profile: Optional[Union[str, float, Dict]] = None,
```

Reply:
Nop, this is accumulated m^3 / 15 mins. I have already converted the water flow profiles in research_scripts repo, and the profiles that you see used in the setup file are m3.